.gitlab-ci.yml

```diff
@@ -0,0 +1,16 @@
+stages:
```

Comment:
I have a deal for you: if you add a CI, you remove one like travis. deal? :)

Reply:
This is not a public one per se, as gitlab runs internally and locally here.